packages/core/admin/admin/src/pages/Webhooks/ListView/index.js

```diff
@@ -267,6 +208,36 @@ function ListView() {
     }
   };
 
+  const handleCheckboxChange = (checkbox, value, id) => {
+    if (checkbox === 'secondary') {
+      dispatch({
+        type: 'SET_WEBHOOKS_TO_DELETE',
+        value,
+        id,
+      });
+    } else {
+      const webhooksIds = webhooks.map(webhook => webhook.id);
+
+      if (webhooksToDelete.length === 0) {
+        for (let i = 0; i < webhooksIds.length; i++) {
+          dispatch({
+            type: 'SET_WEBHOOKS_TO_DELETE',
+            value: true,
+            id: webhooksIds[i],
+          });
+        }
+      } else {
+        for (let i = 0; i < webhooksIds.length; i++) {
+          dispatch({
+            type: 'SET_WEBHOOKS_TO_DELETE',
+            value: false,
+            id: webhooksIds[i],
+          });
+        }
+      }
+    }
```

Comment:
I think this could be simplified:

```js
webhooksIds.forEach(webhook => {
          dispatch({
            type: 'SET_WEBHOOKS_TO_DELETE',
            value: webhooksToDelete.length === 0,
            id: webhook
          })
})
```